api.planx.uk/modules/flows/publish/service.ts

```diff
@@ -48,13 +53,15 @@ export const publishFlow = async (flowId: string, summary?: string) => {
             publisher_id: $publisher_id
             summary: $summary
             has_send_component: $has_send_component
+            is_statutory_application_type: $is_statutory_application_type
           }
         ) {
           id
           flowId: flow_id
           publisherId: publisher_id
           createdAt: created_at
           data
+          is_statutory_application_type
```

Comment:
nit: do we actually need to return `is_statutory_application_type`, are we doing anything with it? Or can we simply _set_ it?